docs/modules/ROOT/pages/includes/quarkus-solace.adoc

```diff
@@ -120,40 +120,6 @@ endif::add-copy-button-to-env-var[]
 |`solace`
 
 
-a| [[quarkus-solace_quarkus.solace.host]]`link:#quarkus-solace_quarkus.solace.host[quarkus.solace.host]`
```

Comment:
@ozangunalp I see that solace specific properties are removed automatically from doc when running quarkus build on my local. I tried to add it again but i see it is removed during build execution. I'm not sure why this is happening?

Reply:
Figured it out. Old package name is still left in target folder which caused the issue. Deleted the target folder manually and triggered quarkus build command. I am able to see the configs in docs folder